File: src/main/java/cn/labzen/web/annotation/Monitor.java
package cn.labzen.web.annotation;

import java.lang.annotation.*;

/**
 * 注解在 Controller 接口或方法上，对指定的API进行指标监控
 * <p>
 * todo 待实现
 */
@Documented
@Retention(RetentionPolicy.SOURCE)
@Target({ElementType.TYPE, ElementType.METHOD})
public @interface Monitor {
}

File: src/main/kotlin/cn/labzen/web/annotation/Monitor.kt
package cn.labzen.web.annotation

/**
 * 注解在 Controller 接口或方法上，对指定的API进行指标监控
 *
 * todo 待实现
 */
@MustBeDocumented
@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION)
@Retention(AnnotationRetention.SOURCE)
annotation class Monitor()
